services/tir.service.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+const grpc = require("@grpc/grpc-js");
+const protoLoader = require("@grpc/proto-loader");
+const path = require("path");
+
+const PROTO_PATH = path.join(__dirname + "/../tir-engine-grpc/proto/tir.proto");
+
+const packageDefinition = protoLoader.loadSync(PROTO_PATH);
+const TirService = grpc.loadPackageDefinition(packageDefinition).tir.TirService;
+
+const client = new TirService(
```

Comment:
Az ilyen típusú klienseket jobb a service `created` methodjában létrehozni, utána már `this.client`-el kell hivatkozni rá a service-ekben, viszont utána a kliens paraméterei mehetnek a service settings-be, vagy központi tárolóba ilyesmi.